<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>User</class>
 <widget class="QDialog" name="User">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_headPortrait">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>91</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/head/images/G.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>添加好友</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>349</x>
     <y>11</y>
     <width>16</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>107</x>
     <y>11</y>
     <width>235</width>
     <height>611</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QLineEdit" name="lineEdit_friendnum"/>
   <widget class="QLineEdit" name="lineEdit_userName"/>
   <widget class="QListWidget" name="listWidget">
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <item>
     <property name="text">
      <string>aaa</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/head/images/A.png</normaloff>:/head/images/A.png</iconset>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_delete">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>删除好友</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
